### PR TITLE
Make —stdout the default export option

### PR DESCRIPTION
**Pull request by manushak**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- remove the `--stdout` flag
- add the `no-output` flag to prevent output to the console
- 2 cases to test 
    - `npm run if-run -- -m ./manifest-path`, it should output the result to the console
    - `npm run if-run -- -m ./manifest-path --no-output`, it should suppress output  to the console

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -32,11 +32,11 @@ export const CONFIG = {
...
         description: '[path to a parameter file that overrides our defaults]',
       },
-      stdout: {
+      'no-output': {
         type: Boolean,
         optional: true,
-        alias: 's',
-        description: '[prints out to the console]',
+        alias: 'n',
+        description: '[privent output to the console]',
```
Comment (narekhovhannisyan) [inline src/config/config.ts:39]:
```suggestion
        description: '[prevent output to the console]',
```

**Review by jmcook1186 — CHANGES_REQUESTED** [review]
This doesn't seem to be working well. Running `if-run` without an output command *does* print the results to the console as expected, however, printing to the console now seems to be the only output option - saving to file is broken.

**Comment by manushak** [comment]
> This doesn't seem to be working well. Running `if-run` without an output command _does_ print the results to the console as expected, however, printing to the console now seems to be the only output option - saving to file is broken.

@jmcook1186, if I understand correctly, now it is working as before (checked in the main branch). When the `outputs: ["yaml"]` is missing from the manifest, the `--output` flag does not work and it does not save. Do I need to change that behavior to save the executed manifest when the `--output` flag with a path is provided, even if `outputs: ["yaml"]` is missing?

**Comment by jmcook1186** [comment]
in my local version adding `outputs: -yaml` in the manifest and using the `-o` command on the command line does not lead to a yaml file being saved, it still just prints o the console. Are you able to save files in the usual way?

**Comment by manushak** [comment]
@jmcook1186 I run `npm run if-run -- -m ./manifests/plugins/divide/success.yml -o ./manifests/outputs/divide` and it works on my local (but beforehand I uncommented `outputs: ["yaml"]`)

**Comment by jmcook1186** [comment]
ok, @manushak it's working for me now too - i must have had something wrong in my setup. Sorry, and thanks!

**Review by narekhovhannisyan — APPROVED** [review]

**Pull request merged**